training/evaluate.py

```diff
@@ -252,13 +252,13 @@ def compute_mr_results(epoch_i, model, eval_loader, opt, criterion=None):
             min_w_l=2, max_w_l=150, move_window_method="left",
             process_func_names=("clip_ts", "round_multiple")
         )
-    elif opt.dset_name == 'charades':
+    elif opt.dset_name in ['charades', 'clotho-moment', 'unav100', 'tut2017']:
         post_processor = PostProcessorDETR(
             clip_length=opt.clip_length, min_ts_val=0, max_ts_val=150,
             min_w_l=2, max_w_l=60, move_window_method="left",
             process_func_names=("clip_ts", "round_multiple")
         )
-    elif opt.dset_name in ['tacos', 'activitynet', 'youtube_highlight', 'clotho-moment', 'unav100', 'tut2017']:
+    elif opt.dset_name in ['tacos', 'activitynet', 'youtube_highlight']:
```

Comment:
Why do you add ['clotho-moment', 'unav100', 'tut2017'] to both of L255 and 261? I think that you need to write only one place.

Reply:
This commit moved ['clotho-moment', 'unav100', 'tut2017'] from L261 to L255 to fix my bug. 
Therefore, the final script does not include ['clotho-moment', ...] in two places, but in only one place.

Reply:
Got it.